18F/identity-idp
# LG-13111: Remove feature flag for email aggregation functionality

## 🎫 Ticket

[LG-13111](https://cm-jira.usa.gov/browse/LG-13111)

## 🛠 Summary of changes

Updates code paths to assume `feature_new_device_alert_aggregation_enabled` as the baseline default behavior.

Recommend reviewing with whitespace changes hidden: https://github.com/18F/identity-idp/pull/11107/files?w=1

## 📜 Testing Plan

Verify no regressions in the behavior of new device email being sent:

1. Sign in from an existing device and observe no email sent
2. Sign in from a new device (e.g. private browsing window) and observe that email is sent after first of either (a) you complete MFA, or (b) 5-10 minutes elapses (depending on when scheduled job runs)
